dockerfiles/incr.sh

```diff
@@ -199,7 +199,7 @@ build_image() {
 
 if [[ -n "$FORCE" ]] ; then
   echo Building local forced build images [ BUILD_LABEL=${BUILD_LABEL} ]
-  build_image platform-build platform-build --build-arg BUILD_USER=${BUILD_USER} --build-arg GITHUB_ACTOR=${BUILD_USER} --build-arg GITHUB_TOKEN=${GITHUB_TOKEN} --build-arg BUILD_TAG=${HEAD} --build-arg BUILD_THREADS=${BUILD_THREADS}
+  build_image platform-build platform-build --build-arg BUILD_USER=${BUILD_USER} --build-arg GITHUB_ACTOR=${BUILD_USER} --build-arg GITHUB_TOKEN=${GITHUB_TOKEN} --build-arg BUILD_TAG=${HEAD} --build-arg BUILD_THREADS=${BUILD_THREADS} --no-cache
```

Comment:
Is this included by mistake?

Reply:
Probably (though arguably it is a good idea to make that change too).